dtm0/service.c

```diff
@@ -524,6 +524,100 @@ M0_INTERNAL void dtm0_service_conns_term(struct m0_dtm0_service *service)
 	M0_LEAVE();
 }
 
+/* -----8<------------------------------------------------------------------- */
+/* CO_FOM service */
+
+struct cfs_service {
+	struct m0_reqh_service cfs_base;
+};
+
+static int cfs_allocate(struct m0_reqh_service           **out,
+			const struct m0_reqh_service_type *stype);
+
+static const struct m0_reqh_service_type_ops cfs_stype_ops = {
+	.rsto_service_allocate = cfs_allocate
+};
+
+struct m0_reqh_service_type m0_cfs_stype = {
```

Comment:
What does this service do?

Reply:
I think this is for executing the co-routines, not used yet.